### PR TITLE
README: remove ../local/.. from auto-completion setup command

### PR DESCRIPTION
**Pull request by CleanMachine1**
I am unsure whether this is correct however I am assuming that most machines  have the ZSH share folder in /usr/share rather than /usr/local/share

### CODE REVIEW

**Comment by MasterOdin** [comment]
I'm not totally sure no that. macOS with homebrew installs all of zsh stuff into `/usr/local/share`, and nothing in `/usr/share`. I think historically that `/usr/local` is the right folder to install third-party stuff that is pertinent to just the local machine and should not be shared over the network.

I would be all for adding a note to say double check the folder location, and depending on your OS, you may need to use `/usr/share`, but I'd like to keep `/usr/local/share` as the default.

**Comment by CleanMachine1** [comment]
From my knowledge of the Linux file system, /usr/bin and or /usr/share is for things installed with the system's package manager, whilst things like packages installed from the AUR for example go into a ./local version

Just a question, does Mac come with ZSH? Or is it a homebrew package

**Comment by CleanMachine1** [comment]
adding a note seems sensible

**Comment by MasterOdin** [comment]
Mac does come with zsh by default. Unfortunately, there's not really any hard rules with file locations (like Homebrew was in `/usr/local/` up until the latest version of macOS, and since then it's in `/opt`), so I don't think there's a "one size fits all" suggestion. It may be worthwhile to start a table to say on these OSes do X and on these other ones do Y and that might be better for everyone?

**Comment by CleanMachine1** [comment]
That idea is good however I think using an entire table for something so simple will over-complicate things, and everyone installs things their own way, so it wouldn't probably be effective anyway

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ If you are experiencing issues with *tldr*, consider deleting the cache files be
 # bash
 tldr --print-completion bash | sudo tee "$BASH_COMPLETION_COMPAT_DIR"/tldr
 # zsh
-tldr --print-completion zsh | sudo tee /usr/local/share/zsh/site-functions/_tldr
+tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr
```
Comment (MasterOdin) [inline README.md:99]: So from the table concept, I think a better way may be something like:

```suggestion
# zsh (recommend double checking where zsh/site-functions directory is located)
# for macOS:
tldr --print-completion zsh | sudo tee /usr/local/share/zsh/site-functions/_tldr

# for Linux:
tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr
```

Comment (CleanMachine1) [reply README.md:99]: It will work however its not a fit all solution.

Comment (MasterOdin) [reply README.md:99]: Well, make whatever change you think is right, I'm done offering suggestions.

Comment (CleanMachine1) [inline README.md:99]:
```suggestion
# zsh (recommend double checking where zsh/site-functions directory is located)
## for macOS:
tldr --print-completion zsh | sudo tee /usr/local/share/zsh/site-functions/_tldr
## for Linux:
tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr
```

File: README.md [hunk 2]
```diff
@@ -96,7 +96,11 @@ If you are experiencing issues with *tldr*, consider deleting the cache files be
 # bash
 tldr --print-completion bash | sudo tee "$BASH_COMPLETION_COMPAT_DIR"/tldr
 # zsh
```
Comment (CleanMachine1) [inline README.md:98]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -95,8 +95,11 @@ If you are experiencing issues with *tldr*, consider deleting the cache files be
 ```bash
 # bash
 tldr --print-completion bash | sudo tee "$BASH_COMPLETION_COMPAT_DIR"/tldr
-# zsh
+# zsh (recommend double checking where zsh/site-functions directory is located)
```
Comment (CleanMachine1) [inline README.md:98]:
```suggestion
# zsh (it is recommended to check where zsh/site-functions directory is located)
```
